Common/UnitDefinitions/Force.json

```diff
@@ -11,6 +11,11 @@
     {
       "SingularName": "Dyn",
       "PluralName": "Dyne",
+      "BaseUnits": {
+        "L": "Centimeter",
```

Comment:
Where do you look up these facts? I have no prior reference. Do you google each you are unsure about or do you have a goto-source? I found this one on wiki.

Reply:
Here's [one ](https://en.wikipedia.org/wiki/Dyne) for dyne:
> The dyne is defined as "the force required to accelerate a mass of one gram at a rate of one centimetre per second per second"

Or try [this](https://www.wolframalpha.com/input/?i=dyne):

> Unit system:
centimeter-gram-second (cgs)

Reply:
Thanks